### ISSUE TITLE
cURL output of HTTPie 

### ISSUE THREAD

**Issue by ryanmcdermott** [jakubroztocil/httpie#325]
I've been a user of HTTPie for a brief bit of time and I haven't seen this feature anywhere, but it might be useful if HTTPie could generate a cURL output of a request. I'm not sure if every particular request could be done in cURL, but this option could be helpful for sharing translatable requests with others that don't have HTTPie.

**Comment by jakubroztocil** [comment]
Yes, it would be a nice thing to have. There are a couple of tools that do something along the lines, but there's no `httpie2curl` at the moment.

* https://github.com/Mashape/httpsnippet
* https://shibukawa.github.io/curl_as_dsl/

**Comment by dzimine** [comment]
+1 for this. We love HTTPie but before the rest of the world speaks HTTPie we need to translate them to cURL. And my cURL is getting rustier the longer I use httpie.

**Comment by sanrodari** [comment]
+1

**Comment by giodamelio** [comment]
:+1:

**Comment by rajadain** [comment]
+1

**Comment by darshan333** [comment]
+1

**Comment by tspike** [comment]
Ooh, yes please!

**Comment by Mgutjahr** [comment]
+1

**Comment by chillenious** [comment]
+1

**Comment by iannickb** [comment]
+1

**Comment by mattconzen** [comment]
+1

**Comment by akrikos** [comment]
Pretty please?

**Comment by cchacin** [comment]
+1

**Comment by Guibod** [comment]
+1

**Comment by Neonox31** [comment]
+1

**Comment by daniel79** [comment]
+1

**Comment by xmavrck** [comment]
+1

**Comment by chalasr** [comment]
+1

**Comment by epayet** [comment]
+1

**Comment by kalbasit** [comment]
@here please stop adding `+1` comments, please use the thumbs up vote on the first post.

**Comment by naartjie** [comment]
This curl command line builder might be useful to anyone looking for an interim solution: https://curlbuilder.com/

**Comment by dcb9** [comment]
Hi guys,
I've just created a toy that is able to convert command arguments between cURL and HTTPie :)
btw thank @defaude that is who let me know there are some people need this feature.

**Comment by jakubroztocil** [comment]
@dcb9 that looks great! 

Links:
* Online tool: http://curl2httpie.online/
* Repo: https://github.com/dcb9/curl2httpie

Screenshot:

<img width="1027" alt="screen shot 2018-04-16 at 11 43 43" src="https://user-images.githubusercontent.com/326885/38823250-7fcdbf70-416b-11e8-9af7-a880e341f93c.png">

**Comment by deronnax** [comment]
what about merging it in httpie with a `--curl` option ?
